loleaflet/src/map/handler/Map.Keyboard.js

```diff
@@ -176,10 +176,17 @@ L.Map.Keyboard = L.Handler.extend({
 		if (container.tabIndex === -1) {
 			container.tabIndex = '0';
 		}
+		
+		L.DomEvent.on(document, 'keydown keyup keypress', this._focusDocument, this);
```

Comment:
Are you sure there might be other situations that we use keyboard & some other element might have the focus, but this overrides that even under readonly permission? Maybe search box or presentation slides might need focus as well

Reply:
yes you are right. Let me correct it.